catalog/ui/catalog-ui-search/src/main/webapp/component/user-settings/user-settings.hbs

```diff
@@ -12,6 +12,7 @@
  **/
  --}}
 <div class="user-settings-navigation">
+    <div class="user-settings-header">Settings</div>
```

Comment:
Instead of writing custom css I would use the `is-header` class.

Reply:
fixed